tests/system/feature_store/test_feature_store.py

```diff
@@ -193,6 +195,8 @@ def test_ingest_and_query(self):
         )  # (*) returns 2 features, label adds 1 feature
         self._get_offline_vector(features, features_size)
 
+        self._get_offline_vector(features, features_size, with_indexes=True)
+
         self._logger.debug("Get online feature vector")
```

Comment:
why there isn't different output for with_indexes=True? or there is and the test is not checking for it?

Reply:
the different output is that the index (ticker) is not showing in the column list, instead it is now the index column.

Reply:
so the test should check for this.

Reply:
but I think there is a way to validate it... on it

Reply:
since now you have a specific test to check the changes you don't need any changes in this test so remove it [lease